.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "build-tools"]
 	path = build-tools
 	url = https://github.com/RedHatInsights/insights-frontend-builder-common.git
+[submodule "_playwright-tests/test-utils"]
+	path = _playwright-tests/test-utils
+	url = git@github.com:content-services/content-sources-backend.git
```

Comment:
i hadn't set up ssh for this project yet and running the test-utils script (and trying to update the submodule manually) got my repo in a very weird state 😅 it might be helpful to add a note about adding a key in order to use this, what do you think?

Reply:
true, this probably should be mentioned somewhere ✍🏼 or should we switch this to https? 🤔 
_(I did this with SSH because for me it's the default way of setting up/using git, might not be the case for everyone 😅 )_

Reply:
yea it got me to finally set up my keys so that's a good thing! 😅 it was easy enough to set up, but not as straightforward to figure out that was what was wrong. the other submodule we have is https, so maybe we should just switch to that? would that break other things? if so, i think adding a note is fine 😺